CHANGELOG.md

```diff
@@ -1,2 +1,3 @@
 # email-split changelog
+1.1.0 - Added "Full" email address into return object
```

Comment:
Please add a newline between this revision and the next one. Otherwise, these will be on the same line when rendered.